src/openrct2/ride/Vehicle.cpp

```diff
@@ -8577,24 +8579,27 @@ void Vehicle::UpdateTrackMotionMiniGolfVehicle(Ride* curRide, rct_ride_entry* ri
 loc_6DCDE4:
     MoveTo(_vehicleCurPosition);
 
-loc_6DCE02:
+    Loc6DCE02(*curRide);
+}
+
+void Vehicle::Loc6DCE02(const Ride& curRide)
+{
     acceleration /= _vehicleUnkF64E10;
     if (TrackSubposition == VehicleTrackSubposition::ChairliftGoingBack)
     {
         return;
     }
+
+    auto trackType = GetTrackType();
+    const auto& ted = GetTrackElementDescriptor(trackType);
+    if (!(std::get<0>(ted.SequenceProperties) & TRACK_SEQUENCE_FLAG_ORIGIN))
     {
-        auto trackType = GetTrackType();
-        const auto& ted = GetTrackElementDescriptor(trackType);
-        if (!(std::get<0>(ted.SequenceProperties) & TRACK_SEQUENCE_FLAG_ORIGIN))
-        {
-            return;
-        }
-        _vehicleMotionTrackFlags |= VEHICLE_UPDATE_MOTION_TRACK_FLAG_3;
-        if (trackType != TrackElemType::EndStation)
-        {
-            return;
-        }
+        return;
+    }
+    _vehicleMotionTrackFlags |= VEHICLE_UPDATE_MOTION_TRACK_FLAG_3;
+    if (trackType != TrackElemType::EndStation)
+    {
+        return;
```

Comment:
Any reason for removing the `{}`? They were adding scope to `trackType` and `ted` which I understand might be an overkill, but seems unrelated to the `goto` change.

Reply:
They probably used to be necessary when it was still a loc, but are not now.